libwallet/src/slate.rs

```diff
@@ -250,8 +250,21 @@ impl Slate {
 		tx
 	}
 
-	/// Create a new slate
+	/// Create a new slate with a plain kernel
 	pub fn blank(num_participants: u8, is_invoice: bool) -> Slate {
+		Slate::blank_with_kernel_features(num_participants, is_invoice, 0, None)
+	}
+
+	/// Create a new slate with custom kernel_feature and lock height
+	/// If a lock_height is set kernel_features will have to be either HeightLocked (2) or NoRecentDuplicate (3)
+	/// otherwise the value passed as lock_height will be ignored and a plain kernel will be created
+	/// Invalid arguments for kernel_features (>3) will be set to 0 (Plain)
```

Comment:
I think we should update this comment as well now that we made it panic

Reply:
very true, thanks! Corrected the comment now.